basm.go

```diff
@@ -61,8 +61,10 @@ type VerifyAttestationInput struct {
 	AcceptableMeasures       []EnclaveMeasurement
 }
 
+type MarshaledAttestedObject []byte
```

Comment:
@matteobjornsson this was borrowed from #4 .  I think once this is in, we should be able to start looking over PR 4 and seeing how these all fit together.